app/utils.py

```diff
@@ -1,14 +1,9 @@
 import string
-import time
 import re
-import aiohttp
-
-from aiohttp.client_exceptions import (ClientConnectionError,
-                                       ClientConnectorError,
-                                       ClientResponseError)
-
-from tenacity import retry, stop_after_attempt, retry_if_exception_message, retry_if_exception_type
+from aiohttp.client_exceptions import (ClientResponseError)
```

Comment:
Import not used, should be removed or fails style guide enforcement.

Reply:
done